recipes/ldas-tools-diskcacheapi-swig/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "ldas-tools-diskcacheAPI-swig" %}
+{% set version = "2.6.6" %}
+
+# dependencies
+{% set ldas_tools_cmake_version = "1.0.4" %}
+{% set ldas_tools_diskcacheapi_version = "2.6.2" %}
+{% set ldas_tools_ldasgen_swig_version = "2.6.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz"
+  sha256: b5d2f444d1ef621d93253aa20ed29c4cb71531e34c6afaca4d1041f09cf83de2
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.6
+    - pkg-config  # [not win]
+    - make  # [not win]
+    - swig 3
+  host:
+    - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+    - ldas-tools-diskcacheapi >={{ ldas_tools_diskcacheapi_version }}
+  run:
+    - ldas-tools-diskcacheapi >={{ ldas_tools_diskcacheapi_version }}
```

Comment:
need outputs here

Reply:
@scopatz, are you just asking me to move the top-level `test` section to below the `outputs` section, or do you mean something else?